packages/ndla-ui/src/Footer/Footer.tsx

```diff
@@ -147,7 +148,9 @@ const Footer = ({ children, commonLinks, links, languageSelector, auth, privacyL
             <StyledFooterHeaderIcon />
           </div>
           <div>
-            <StyledHeader>{t("footer.vision")}</StyledHeader>
+            <StyledHeader element="div" textStyle="content-alt" margin="none">
```

Comment:
Dette er jo berre en tekst så antar span er meir korrekt.
```suggestion
            <StyledHeader element="span" textStyle="content-alt" margin="none">
```

Reply:
Det vil påvirke margins. Kan evt wrappe den med en div for å få riktig styling men vil det være "cleanere"?

Reply:
Nei, sikkert ikkje. Kan sikkert la det stå.